.vsts-ci/templates/nanoserver.yml

```diff
@@ -0,0 +1,62 @@
+parameters:
```

Comment:
@TravisEz13, your last commit had 1 failures in `PowerShell-CI-windows`
CmsMessage cmdlets and Get-PfxCertificate basic tests.Verify a CMS message can be protected / unprotected
```powershell
Keyset does not exist
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\CmsMessage.Tests.ps1: line 71
71:         $decrypted = $message | Unprotect-CmsMessage -To $certLocation

```